modules/interacting-serverless-apps-http2-gRPC-up-to-4-9.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// serverless/knative-serving/external-ingress-routing/using-http2-gRPC.adoc
+
+:_content-type: PROCEDURE
+[id="interacting-serverless-apps-http2-gRPC-up-to-4-9_{context}"]
+= Interacting with a serverless application using HTTP2 and gRPC in OpenShift Dedicated 4.9 and older
```

Comment:
```suggestion
= Interacting with a serverless application using HTTP2 and gRPC in {product-title} Dedicated 4.9 and older
```

Reply:
@skrthomas applied your suggestion as such:
`in {product-title} 4.9`  (results into in _OpenShift Dedicated 4.9_)
if we don't omit `Dedicated`, we get it twice:
`in {product-title} Dedicated 4.9`  (results into in _OpenShift Dedicated Dedicated 4.9_)